### PR TITLE
feat/add install script

### PR DESCRIPTION
**Pull request by colek42**
This adds an install script, updates the README, and moves examples into the witness-examples repo.

### CODE REVIEW

**Comment by colek42** [comment]
bash <(curl -s https://raw.githubusercontent.com/testifysec/witness/install-script/install-witness.sh)

File: install-witness.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+if [ $? -ne 0 ]; then
+    echo "Checksum verification failed, exiting"
+    exit 1
+fi
+
+# Extract and install the binary
+tar -xzf witness_${VERSION}_${OS}_${ARCH}.tar.gz
+
+# Move the binary to /usr/local/bin
+sudo mv witness /usr/local/bin/
```
Comment (mikhailswift) [inline install-witness.sh:67]: Should this script support Windows? The line above indicates windows is a supported environment but these steps will not work on Windows.

**Comment by colek42** [comment]
@mikhailswift can you make sure this works on darwin, as I am unable to test

**Comment by mikhailswift** [comment]
works on amc

**Review by mikhailswift — APPROVED** [review]

**Pull request merged**